.github/workflows/BuildAndDeploy.yaml

```diff
@@ -34,14 +30,10 @@ jobs:
           persist-credentials: false
           path: docs
 
-      - name: Install docs deps
-        run: yarn --cwd ./docs install
-      - name: Generate docs api references
-        run: yarn --cwd ./docs generate-api-reference
       - name: Build docs site with antora
-        uses: kameshsampath/antora-site-action@master
-        with:
-          antora_playbook: docs/antora-playbook.yml
+        run: |
+          yarn add @antora/lunr-extension
```

Comment:
I don't think this is necessary. If you run this command locally in `/docs`, the package will automatically be installed when `yarn install` is executed.